bin/grab-grub-efi

```diff
@@ -59,7 +59,10 @@ deb2tar() {
     debug 1 "creating $out in tempdir at $tdir"
 
     mkdir -p "$tdir/deb"
-    dpkg -x "$deb" "$tdir/deb" || fail "failed to unpack $deb"
+    ar x "$deb" --output "$tdir/deb" || fail "failed to unpack $deb"
+    cd "$tdir/deb"
+    tar -xf data.tar.*  # should handle *.gz/xz/zst
```

Comment:
you're not checking error n this. or the either of the 'cd'.

Avoid the cd - by doing:

```
( cd "$tdir/deb" && tar -xf data.tar.* ) || fail "failed to extract data"
```

Reply:
Changed in all places. Hope this time it is better.